app/src/main/java/cs5150athletetracking/com/athletetracking/ResultCallable.java

```diff
@@ -0,0 +1,10 @@
+package cs5150athletetracking.com.athletetracking;
+
+/**
+ * Created by mag94 on 10/20/15.
```

Comment:
I merged my result callable into staging. If it isn't showing up for you on your branch, pull in from staging.